.github/workflows/README.md

```diff
@@ -0,0 +1,83 @@
+# DevOps Apprenticeship: Project Exercise
```

Comment:
Did you mean to add this file under the workflows folder?

Reply:
deleted :)